README.md

```diff
@@ -167,6 +167,7 @@ To build SGX-LKL in release mode, run:
 
 4. To install SGX-LKL on the host system, use the following command:
 ```
+source /opt/openenclave/share/openenclave/openenclaverc
```

Comment:
Any reason you added this here? It's also needed for all the build commands and is mentioned in line 117 already.

Reply:
I added it since it doesn't hurt running again and if current shell is lost for any reason it is not easy to figure out that they need to run this command again. So I think we should add this line so that it works regardless. Will be easier for users. I experiencend myself several times that need to run this command again.